lib/generators/boring/vcr/install/install_generator.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module Boring
+  module Vcr
+    class InstallGenerator < Rails::Generators::Base
+      desc "Adds VCR to the application"
+      source_root File.expand_path('templates', __dir__)
+
+      class_option :test_framework,
+                   type: :string,
+                   alias: "-f",
+                   default: "rspec",
+                   enum: %w[rspec minitest],
+                   desc: "Tell us which test framework you are using. Default to rspec"
+
+      class_option :stubbing_libraries,
+                   type: :array,
+                   alias: "-sl",
+                   default: ['webmock'],
+                   enum: %w[webmock typhoeus faraday excon],
+                   desc: "Tell us stubbing library you want to use seprated by space. Default to webmock"
+
+      def add_vcr_gem
+        say "Adding VCR gems to Gemfile", :green
+        # TODO: Use check_and_install_gem method when it is available
+        gem 'vcr', group: :test
+      end
+
+      def add_stubbing_library_gems
+        say "Adding stubbing library gems to Gemfile", :green
+
+        options[:stubbing_libraries].uniq.each do |stubbing_library|
+          # TODO: Use check_and_install_gem method when it is available
+          gem stubbing_library, group: :test
+        end
+      end
+
+      def setup_vcr_for_rspec
+        test_framework = options[:test_framework]&.to_s || 'rspec'
+        return unless test_framework == 'rspec'
+
+        say "Setting up VCR for RSpec", :green
+
+        @stubbing_libraries = format_stubbing_libraries
+
+        template("vcr.rb", "spec/support/vcr.rb")
+
+        inject_into_file "spec/rails_helper.rb",
```

Comment:
Normally all support files are loaded with:

```
# The following line is provided for convenience purposes. It has the downside
# of increasing the boot-up time by auto-requiring all files in the support
# directory. Alternatively, in the individual `*_spec.rb` files, manually
# require only the support files necessary.
#
Dir[Rails.root.join('spec/support/**/*.rb')].sort.each { |f| require f }
```

So we don't need separate require, let's check that line and only add require if it doesn't exist

Reply:
it seems new rspec-rails add this line "Rails.root.glob('spec/support/**/*.rb').sort.each { |f| require f }"
so I will check for this line